src/test_workflow/benchmark_test/benchmark_args.py

```diff
@@ -50,117 +50,162 @@ class BenchmarkArgs:
+            self.password = args.password if args.password else None
+            self.manager_node_count = args.manager_node_count if args.manager_node_count else None
+            self.data_node_count = args.data_node_count if args.data_node_count else None
+            self.client_node_count = args.client_node_count if args.client_node_count else None
+            self.ingest_node_count = args.ingest_node_count if args.ingest_node_count else None
+            self.ml_node_count = args.ml_node_count if args.ml_node_count else None
+            self.jvm_sys_props = args.jvm_sys_props if args.jvm_sys_props else None
+            self.data_node_storage = args.data_node_storage if args.data_node_storage else None
+            self.ml_node_storage = args.ml_node_storage if args.ml_node_storage else None
+            self.enable_remote_store = args.enable_remote_store
+            self.data_instance_type = args.data_instance_type if args.data_instance_type else None
+            self.workload = args.workload
+            self.workload_params = args.workload_params if args.workload_params else None
+            self.test_procedure = args.test_procedure if args.test_procedure else None
+            self.exclude_tasks = args.exclude_tasks if args.exclude_tasks else None
+            self.include_tasks = args.include_tasks if args.include_tasks else None
+            self.benchmark_config = args.benchmark_config if args.benchmark_config else None
+            self.user_tag = args.user_tag if args.user_tag else None
+            self.additional_config = json.dumps(args.additional_config) if args.additional_config is not None else None
+            self.use_50_percent_heap = args.use_50_percent_heap
+            self.telemetry = args.telemetry
+            self.telemetry_params = args.telemetry_params if args.telemetry_params else None
+            self.logging_level = args.logging_level
 
-        if self.bundle_manifest is None and self.distribution_url is None and self.cluster_endpoint is None:
-            raise Exception('Please provide either --bundle-manifest or --distribution-url  or --cluster_endpoint to run the performance test.')
-        elif self.distribution_url and self.distribution_version is None:
-            raise Exception("--distribution-version is required parameter while using --distribution-url param.")
+            if self.bundle_manifest is None and self.distribution_url is None and self.cluster_endpoint is None:
+                raise Exception("Please provide either --bundle-manifest or --distribution-url  or --cluster_endpoint to run the performance test.")
+            elif self.distribution_url and self.distribution_version is None:
+                raise Exception("--distribution-version is required parameter while using --distribution-url param.")
+        elif args.command == "compare":
+            self.baseline = args.baseline if hasattr(args, "baseline") else None
```

Comment:
raise exception if either `baseline` or `contender` value is not provided as they are mandatory arguments for OSB compare command.